### ISSUE TITLE
Task types

### ISSUE THREAD

**Issue by olantrust** [humhub/humhub-modules-tasks#130]
1) Ability to set types 

2) Ability to assign tasks to a type
e.g. :  Issue, Repair, Feedback, Admin, Complaint

3) Ability to filter tasks by type

**Comment by luke-** [comment]
Here it might be good to use the existing Topic System.

**Comment by luke-** [comment]
https://github.com/humhub/humhub-modules-tasks/issues/132

**Comment by olantrust** [comment]
Agreed topics are good here; however tasks types cover a different use case - for example:
User opens a task which is a 'complaint' and marks it as such. 
The task relates to 'finance' thus it is marked as that topic.

I guess it could either:
a) Use the topics system
b) Use a simple drop-down choice (variables defined in settings) the benefit of that is if eventually different form templates were to be used for different task types...
